### ISSUE TITLE
Integrate UI Cognito flow with backend Cogntio

### ISSUE THREAD

**Issue by brettstack** [RareCamp/RareCamp#24]
Currently the UI is mocked out, we need to make calls to the actual AWS resources using Amplify APIs (wrappers around cognito).

**Comment by brettstack** [comment]
This is blocked on getting Vysakh access to the AWS resources

**Comment by brettstack** [comment]
Duplicate of #7
